Makefile

```diff
@@ -1,3 +1,13 @@
+install:
+	npm install jspm --save-dev
+	jspm init
+	jspm install core-js
+	jspm install react
+	jspm install react-dom
+	jspm install fetch
+	jspm dl-loader --latest
+	npm install uglifyjs
+	npm install html-dist
```

Comment:
all these dependencies should be in the package.json and then the install is just `npm install && ./node_modules/.bin/jspm install`